my-react-app/src/components/preloader/Preloader.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import './Preloader.css';
+
+export function Preloader() {
+  return (
+    <div className="preloader__container">
+      <div className="lds-ellipsis">
+        <div></div>
```

Comment:
you should not use empty `div`s
```suggestion
        <div className="scale-up-ellips"/>
```